test/functional/plugin/lsp/config_callbacks_spec.lua

```diff
@@ -25,18 +26,41 @@ describe('LSP Callback Configuration', function()
 
           assert(a == b)
         end
+
+        table_key_length = function(table)
+          local callback_length = 0
+          for _k, _v in pairs(table) do
+            callback_length = callback_length + 1
+          end
+          return callback_length
+        end
       EOF
     ]]))
   end)
 
+  it('should have no default callback', function()
+    lua[[eq(table_key_length(lsp_callbacks._callback_mapping), 0)]]
```

Comment:
This example don't work well. I want to know how to debug lua functional test.
If I put like this `lua[[inspect(table_key_length(lsp_callbacks._callback_mapping))]]`, it doesn't put output.

That example said like this.
```sh
[  ERROR   ] test/functional/plugin/lsp/config_callbacks_spec.lua @ 41: LSP Callback Configuration should have no default callback
test/functional/helpers.lua:96: Error executing lua: [string "<VimL compiled string>"]:11: assertion failed!

stack traceback:
	test/functional/helpers.lua:96: in function 'request'
	test/functional/helpers.lua:259: in function 'lua'
	test/functional/plugin/lsp/config_callbacks_spec.lua:42: in function <test/functional/plugin/lsp/config_callbacks_spec.lua:41>
```

Reply:
If I understand what you want to do correctly, you should look into `nvim_execute_lua`.

Reply:
In the tree-sitter branch I added the following helper, feel free to copy it:
```
local function exec_lua(code, ...)
  return meths.execute_lua(code, {...})
end
```

Then you can use the `exec_lua[[ ... ]]` syntax

Reply:
Ah I see this branch already did that, I would still rename it to `exec_lua`, so it is clear what context is being used.

Reply:
Thanks :-) @KillTheMule @bfredl 
[memo]
https://github.com/neovim/neovim/pull/10124/files#diff-33008990d305461ece636fb5f89a6b7bR689